### PR TITLE
feat: added questions and results functionality

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Created basic functionalities like 
Answering quiz and results page with local data
As of now only the first category is wired please check only with that
https://asimov-quiz.netlify.app/

### CODE REVIEW

File: src/pages/Questions.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
```
Comment (gopalbbharadva) [inline src/pages/Questions.tsx:3]: Can be in one import.

File: public/index.html [hunk 2]
```diff
@@ -39,5 +50,5 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
```
Comment (gopalbbharadva) [inline public/index.html:52]: you can remove these default comments.

**Review by gopalbbharadva — APPROVED** [review]
Code looks fine and well- structured. Good to go.

**Review by romabulani — APPROVED** [review]
- Liked that you have used different folder for types 
- Clean and Modular code
- Good to merge

**Pull request merged**